partner_second_lastname/tests/__init__.py

```diff
@@ -1,6 +1,5 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from . import test_name
-from . import test_onchange
```

Comment:
Why removing these tests?

Reply:
There were an issue with it and I think it is not needed
It was also removed in #805